### PR TITLE
Break up misc-ro.sh into separated cases

### PR DESCRIPTION
**Pull request by HuijingHei**
Break up misc-ro.sh into separated cases

### CODE REVIEW

**Comment by dustymabe** [comment]
I think we need more context here in the git commit message.. I *think* I know the answer to these questions but most people might not:

- What misc-ro.sh are you referring to?
- Where did these tests come from?

**Comment by HuijingHei** [comment]
> I think we need more context here in the git commit message.. I _think_ I know the answer to these questions but most people might not:
> 
> * What misc-ro.sh are you referring to?
> * Where did these tests come from?

Thanks @dustymabe for your review
misc-ro.sh is from https://github.com/openshift/os/blob/master/tests/kola/misc-ro/misc-ro.sh
This PR is shared cases with fcos and rhcos, rhcos only PR is https://github.com/openshift/os/pull/680

**Comment by dustymabe** [comment]
> Thanks @dustymabe for your review

No Problem! :) 

> misc-ro.sh is from https://github.com/openshift/os/blob/master/tests/kola/misc-ro/misc-ro.sh This PR is shared cases with fcos and rhcos, rhcos only PR is [openshift/os#680](https://github.com/openshift/os/pull/680)

Right, but the git commit message doesn't say any of that.. Here's an example of what the git commit message should probably look like:

```
tests: import test cases from openshift/os repo

These are tests cases that were in `misc-ro.sh` in the openshift/os repo [1].
We're importing them here to share them between FCOS and RHCOS because
they benefit both platforms.

[1] https://github.com/openshift/os/blob/43044b95eb9288162dd62b84ad7dc7b6c8b5707a/tests/kola/misc-ro/misc-ro.sh
```

File: tests/kola/networking/dhcpclient [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (dustymabe) [inline tests/kola/networking/dhcpclient]: I would maybe rename this file `nm-dhcp-client`

File: tests/kola/files/selinux [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (dustymabe) [inline tests/kola/files/selinux]: let's create a new `tests/kola/selinux` dir and put this test under `tests/kola/selinux/enforcing`

File: tests/kola/files/usrlocal-selinux [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (dustymabe) [inline tests/kola/files/usrlocal-selinux]: let's create a new `tests/kola/selinux` dir and put this test under `tests/kola/selinux/usrlocal-context`

**Comment by HuijingHei** [comment]
Done, thanks!

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**